.github/policies/labelAdded.highestVersionRemoval.yml

```diff
@@ -20,13 +20,10 @@ configuration:
         then:
           - addReply:
               reply: >-
-                Hello @${issueAuthor},
+                Hi @${issueAuthor},
 
 
-                This PR removes the highest available version of the package from the repository.
-
-
-                Please provide the reason for removal to help our moderators understand the context.
```

Comment:
Can we keep this portion of the message? There are several cases where users submit removals without putting anything in the body to indicate why it is being removed